test/integration/targets/s3_bucket/tasks/main.yml

```diff
@@ -0,0 +1,182 @@
+---
+
+- block:
+
+    # ============================================================
+    - name: set connection information for all tasks
+      set_fact:
+        aws_connection_info: &aws_connection_info
+          aws_access_key: "{{ aws_access_key }}"
+          aws_secret_key: "{{ aws_secret_key }}"
+          security_token: "{{ security_token }}"
+          region: "{{ aws_region }}"
+      no_log: true
+
+    # ============================================================
+    - name: Create simple s3_bucket
+      s3_bucket:
+        name: "{{ resource_prefix }}-testbucket-ansible"
```

Comment:
Anywhere that's using resource_prefix in the bucket name needs to be changed to {{ resource_prefix | lower }} because resource_prefix is not necessarily lowercase (such as, on a mac it may be ansible-test-shertel-OSX-12345678 which fails). Would also be good to have a test that a bucket's policy can be modified.

Reply:
My view is that resource_prefix should be fixed at source rather than here.

Reply:
Done. 

Thank you for the CI updates.

Reply:
@willthames I only saw your comment after I pushed my changes

Reply:
Thanks @willthames